### ISSUE TITLE
support basic numpy types

### ISSUE THREAD

**Issue by ct-clmsn** [STEllAR-GROUP/phylanx#501]
numpy has internal data structures that might require some support or mapping into phylanx types.

documentation can be found here: https://docs.scipy.org/doc/numpy-1.13.0/user/basics.types.html

**Comment by hkaiser** [comment]
@ct-clmsn yes, you make a valid point. Currently we support three types: `bool`, `int64`, and `double`. All other numpy types (`int32` etc.) are explicitly converted. We clearly miss complex, though.
